mmcv/runner/base_runner.py

```diff
@@ -414,12 +414,23 @@ def register_logger_hooks(self, log_config):
                 info, HOOKS, default_args=dict(interval=log_interval))
             self.register_hook(logger_hook, priority='VERY_LOW')
 
+    def register_itertimer_hook(self, itertimer_config):
+        if itertimer_config is None:
+            return
+        if isinstance(itertimer_config, dict):
+            itertimer_config.setdefault('type', 'IterTimerHook')
+            hook = mmcv.buid_from_cfg(itertimer_config, HOOKS)
+        else:
+            hook = itertimer_config
+        self.register_hook(hook)
+
     def register_training_hooks(self,
                                 lr_config,
                                 optimizer_config=None,
                                 checkpoint_config=None,
                                 log_config=None,
-                                momentum_config=None):
+                                momentum_config=None,
+                                itertimer_config=None):
```

Comment:
This modification will change the default behavior.

Reply:
Thanks for your comments. I'll set a default value for itertimer_config.